recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "1.4.0" %}
+{% set version = "2.1.0" %}
```

Comment:
2.1 is the latest version of AllenNLP, but we're still releasing from the 1.x branch occasionally. Switching between the versions like this seems like it would be a problem?

Reply:
In principle you're right; partially for that reason, I created a 1.x branch. This PR is for master, which however is still at 1.4 as well since #16 hasn't been merged yet due to being blocked on https://github.com/conda-forge/torchvision-feedstock, which itself is blocked on CI woes (no GPU, the underlying problem coming from the pytorch-feedstock which needs to be build manually due to timeouts, etc.). I'm trying to solve this in the background as well, but for the moment we're stuck.